employees/views.py

```diff
@@ -154,6 +156,80 @@ def redirect_to_current_month(self) -> HttpResponseRedirectBase:
         return redirect(self._get_recent_month_url(pk))
 
 
+@method_decorator(login_required, name="dispatch")
+@method_decorator(
+    check_permissions(
+        allowed_user_types=[
+            CustomUser.UserType.EMPLOYEE.name,
+            CustomUser.UserType.MANAGER.name,
+            CustomUser.UserType.ADMIN.name,
+        ]
+    ),
+    name="dispatch",
+)
+class ReportListCreateProjectJoinView(MonthNavigationMixin, CreateView):
+    template_name = "employees/report_list.html"
+    project_join_form = ProjectJoinForm
+    model = Report
+    form_class = ReportForm
+    url_name = "custom-report-list"
+    object = None
+
+    def get_initial(self) -> dict:
+        initial = super().get_initial()
+        initial.update({"date": timezone.now(), "user": self.request.user})
```

Comment:
Why do you need `date` in initial data?

Reply:
to prepopulate the form with `today`. It was my impression that was the way the old view has been nahdling the form, so I wanted to keep it.